lib/core/include/irods/irods_path_recursion.hpp

```diff
@@ -44,12 +44,12 @@ namespace irods
     // This is what the icommand xxxxUtil() function uses to scan recursively
     // for directories/symlinks. The path is the user's non-canonical path, and
     // the recursion_map_t is statically defined by the caller as needed.
-    // The bool parameter is true if the "--link" flag was specified
+    // The bool parameter is true if the "--ignore-symlinks" flag was specified
     int check_directories_for_loops( boost::filesystem::path const &, irods::recursion_map_t &, bool);
```

Comment:
Please rephrase L47 so it makes sense.

Reply:
See https://github.com/irods/irods/pull/8323#discussion_r2014397655.

Reply:
Done.